scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -107,5 +107,63 @@ Use the *rhel-tools* container with the following:
 
 See the link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/getting-started-with-containers/chapter-11-using-the-atomic-tools-container-image[RHEL Tools Container documentation] for more information.
 
+[[scaling-performance-debugging-using-oa-image]]
+== Debugging {product-title} Using the OpenShift-Ansible Image
+
+Red Hat distributes an https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[openshift-ansible image], with specific checks focused on detecting common deployment issues.
+Use the following checks to help detect potential issues:
+
+[[diagnostic-checks]]
+.Diagnostic Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
+
+|`*etcd_imagedata_size*`
+|This check measures the total size of OpenShift image data in an etcd cluster.
+Fails if the calculated size exceeds a user-defined limit. If no limit is specified, this check will fail if the size of OpenShift image data exceeds a certain amount of the currently used space in the etcd cluster.
+
+A failure from this check indicates that a significant amount of space in etcd is being taken up by OpenShift image data, which can destabilize an etcd cluster.
+
+A user-defined limit may be set by passing the variable `etcd_max_image_data_size_bytes=40000000000` to the `openshift_health_checker` role.
+This example limit will cause the check to fail if the total size of OpenShift image data stored in etcd exceeds `40GB`.
+
+A user-defined value may be set for this variable by passing it as an option to the role:
+
+`# ansible-playbook -i /etc/ansible/hosts playbooks/common/openshift-checks/check.yml -e etcd_max_image_data_size_bytes=40000000000`
+
+It may also be passed as part of the `OPTS` variable, if running the playbook through the Docker image:
+
+`# docker run ... -e OPTS="-v -e etcd_max_image_data_size_bytes=40000000000"`
+
+See below for a complete example of running checks with the Docker image.
+
+|`*etcd_traffic*`
+|This check detects higher-than-normal traffic on an etcd host. Fails if a `journalctl` log entry with an etcd sync duration warning is found.
+
+For further information on improving etcd performance, see the link:host_practices.adoc[Host Practices documentation].
+
+|===
+
+
+Use the *openshift-ansible* diagnostic checks with the following:
+
+----
+# docker run -u `id -u` \
+       -v $HOME/.ssh/id_rsa:/opt/app-root/src/.ssh/id_rsa:Z,ro \
+       -v /etc/ansible/hosts:/tmp/inventory:ro \
+       -e INVENTORY_FILE=/tmp/inventory \
+       -e OPTS="-v" \
+       -e PLAYBOOK_FILE=playbooks/byo/openshift-preflight/check.yml \
```

Comment:
Done. Added a mention of the [logging_index_time](https://github.com/openshift/openshift-ansible/pull/4682) check, and linked to the existing [Additional Diagnostics Checks...](https://github.com/openshift/openshift-docs/pull/4511/files#diff-eb951a90abca9ed6bf4a7cb864087988) section in the `diagnostics_tool` docs